json-schema/json-schema-circe/src/main/scala/endpoints/circe/JsonSchemas.scala

```diff
@@ -5,6 +5,7 @@ import endpoints.algebra.circe.CirceCodec
 import io.circe._
```

Comment:
It seems that the changes in this file do nothing. Would it be possible to revert them?

Reply:
Sure I can revert it. This happened because of adding `orElseMerge` then formatting, and then removing `orElseMerge` again because it became unnecessary. (the changes are inline with the scalafmt conf though)